src/acopf_admm_gpu.jl

```diff
@@ -251,146 +251,254 @@ function check_linelimit_violation(data, u::Array{Float64,1})
     return rateA_nviols, rateA_maxviol, rateC_nviols, rateC_maxviol
 end
 
-function admm_rect_gpu(case; iterlim=800, rho_pq=400.0, rho_va=40000.0, scale=1e-4,
-                       use_gpu=false, use_polar=true, gpu_no=1)
-    data = opf_loaddata(case)
-
-    ngen = length(data.generators)
-    nline = length(data.lines)
-    nbus = length(data.buses)
-    nvar = 2*ngen + 8*nline
+"""
+    Parameters
+
+This contains the parameters used in ADMM algorithm.
+"""
+mutable struct Parameters
```

Comment:
Nice! What do you think of using `Base.@kwdef` to define directly the default parameters?